egcg_core/util.py

```diff
@@ -97,5 +89,6 @@ def move_dir(src_dir, dest_dir):
                 dest_file = os.path.join(dest_dir, os.path.basename(fp))
                 shutil.move(fp, dest_file)
         return 0
-    except OSError:
+    except OSError as e:
+        app_logger.error('OSError: ' + str(e))
```

Comment:
We're now logging `OSError`s here, but should we be catching them in the first place?